design/sketch.webidl

```diff
@@ -419,7 +419,6 @@ dictionary WebGPUAttachmentsState {
 };
 
 // Common stuff for ComputePipeline and RenderPipeline
-typedef u32 WebGPUShaderStageEnum;
 interface WebGPUShaderStage {
```

Comment:
Sorry this need to be deleted as well, both the typedef and the interface were to declare the same enum

Reply:
So this WebGPUShaderStage is no longer need anymore ? Got it.